### PR TITLE
Added capabilities to launch via docker-compose

### PR DESCRIPTION
**Pull request by umed**
- [x] Added Dockerfiles for frontend and backend
- [x] Added Dockerfile to initialise Postgres database
- [x] Fixed config.examples due to required password by Postgres if deployed on different hosts
- [x] Added logging in nginx_example.conf
- [ ] Added docker-compose.yml

### CODE REVIEW

**Comment by umed** [comment]
Hey there,

I prepared draft to allow users/developers launch project via docker-compose.
Thought it would be better if I show some draft and get feedback then finish it.

**Comment by jadeydi** [comment]
Thanks for your PR, Golang is simple for deploy, but I think docker is ok for me.

File: deploy/nginx_example.conf [hunk 1]
```diff
@@ -4,6 +4,10 @@ upstream satellity_http_server {
 
 server {
   listen 80;
+
+  access_log /var/log/nginx/access.log;
+  error_log /var/log/nginx/error.log;
```
Comment (jadeydi) [inline deploy/nginx_example.conf:9]: Nginx has default log config, I don't think we need this

Comment (umed) [reply deploy/nginx_example.conf:9]: ok, I will confirm

**Pull request closed without merging**